scholarly/_navigator.py

```diff
@@ -246,6 +268,9 @@ def _get_soup(self, url: str) -> BeautifulSoup:
         html = html.replace(u'\xa0', u' ')
         res = BeautifulSoup(html, 'html.parser')
         try:
+            # @ipeirotis: I do not understand the usage of publib
```

Comment:
If you don't have publib you cannot collect the publication bib.
I think it could be moved elsewhere as well, but this was the best ad-hoc solution I found so far.